arrow-optics/src/main/kotlin/arrow/optics/Fold.kt

```diff
@@ -23,13 +10,12 @@ import arrow.typeclasses.Monoid
  * @param S the source of a [Fold]
  * @param A the target of a [Fold]
  */
-@higherkind
-interface Fold<S, A> : FoldOf<S, A> {
+interface Fold<S, A> {
```

Comment:
Can this be made a `fun interface` if we move `R` to the type signature generics?